test/buildkite/install-gcp.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+# Run as root.
+
+sh -c 'echo deb https://apt.buildkite.com/buildkite-agent stable main > /etc/apt/sources.list.d/buildkite-agent.list'
+apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 32A37959C2FA5C3C99EFBC32A79206696452D198
+apt-get update && sudo apt-get install -y buildkite-agent
+sed -i "s/xxx/${BUILDKITE_TOKEN}/g" /etc/buildkite-agent/buildkite-agent.cfg
+
+systemctl enable buildkite-agent && sudo systemctl start buildkite-agent
+
+usermod -a -G docker buildkite-agent
```

Comment:
should add builddkite-agent to the docker group prior to enabling in systemctl.

Reply:
Right. The script is based on the history - I think I restarted the agent after. Will update it.